src/Tools/Extensions.ApiDescription.Client/src/build/Microsoft.Extensions.ApiDescription.Client.targets

```diff
@@ -84,6 +84,7 @@
 
   <Target Name="_InnerGenerateOpenApiCode" DependsOnTargets="_GetCurrentOpenApiReference;$(GeneratorTarget)" />
 
+  <!-- Note target will **always** execute when generator uses the OutputPath as a directory. -->
```

Comment:
Noticed while testing other changes. After an email conversation w/ `msbuild` team, I doubt this will change because directory timestamps aren't updated when files in them are added or changed but docs might be improved.

Reply:
/fyi @darrelmiller